src/index.js

```diff
@@ -1,10 +1,11 @@
 import { dirname } from 'path';
-import { buildExternalHelpers, transform } from 'babel-core';
+import { buildExternalHelpers, transform, version as babelVersion } from 'babel-core';
 import { createFilter } from 'rollup-pluginutils';
 import preflightCheck from './preflightCheck.js';
 import { warnOnce } from './utils.js';
 import { RUNTIME, BUNDLED, HELPERS } from './constants.js';
 
+const babelsMajor = parseInt(babelVersion, 10);
```

Comment:
personally I believe supporting 2 versions of babel here just creates unnecessary mess in the code and that we should just release new major (some prerelease for now, until babel7 gets stable) with a support for babel7 exclusively, wdyt @hzoo ?